app/scripts/controllers/transactions/index.js

```diff
@@ -1090,4 +1072,43 @@ export default class TransactionController extends EventEmitter {
       }
     }
   }
+
+  _trackTxEvent(txMeta, event, extraOpts = {}) {
+    const {
+      type,
+      status,
+      chainId,
+      origin: referrer,
+      txParams: { gasPrice },
+      metamaskNetworkId: network,
+    } = txMeta;
+    const source = referrer === 'metamask' ? 'user' : 'dapp';
+
+    this._trackMetaMetricsEvent({
+      event,
+      category: 'Transactions',
+      sensitiveProperties: {
+        type,
+        status,
+        gasPrice,
+        referrer,
+        source,
+        network,
+        chain_id: chainId,
```

Comment:
let's add 'first_seen:' and use txMeta.time (i think) which is only set when the tx is first seen

Reply:
done